### PR TITLE
fix(tests): Make runner more stable & add update process testing

### PR DESCRIPTION
**Pull request by smalluban**
* Clears out the self-opening onboarding page after extension installation - it made some test flaky
* Moves config files into the `e2e` path and specs to `spec` subdirectory - as the file number grows, it is more readable when everything is in the `tests/e2e` scope
* Renames specs, so we can use glob pattern for future use without changing the config
* Adds `npm run wdio:update` command to test suite with updated extension from the latest release

### CODE REVIEW

File: tests/e2e/spec/_onboarding.spec.js [hunk 1]
```diff
@@ -19,9 +23,11 @@ describe('Onboarding', function () {
     await getExtensionElement('button:skip').click();
     await expect(getExtensionElement('view:skip')).toBeDisplayed();
 
-    await browser.url(await getExtensionPageURL('panel'));
+    await switchToPanel(async function () {
+      await expect(getExtensionElement('button:enable')).toBeDisplayed();
+    });
 
-    await expect(getExtensionElement('button:enable')).toBeDisplayed();
+    await browser.url('about:blank');
```
Comment (chrmod) [inline tests/e2e/spec/_onboarding.spec.js:30]: maybe put it in `afterEach`?

File: tests/e2e/utils.js [hunk 2]
```diff
@@ -24,18 +24,20 @@ async function getExtensionId() {
   if (!extensionId) {
     switch (browser.capabilities.browserName) {
       case 'chrome': {
-        await browser.url('chrome://extensions');
+        const url = 'chrome://extensions';
+        await browser.url(url);
```
Comment (chrmod) [inline tests/e2e/utils.js:28]: why we need a variable ?

**Review by chrmod — COMMENTED** [review]
First time I see extension update tests. Really cool stuff @smalluban !

File: tests/e2e/utils.js (hunk 2 above)
Comment (smalluban) [reply tests/e2e/utils.js:28]: It must be a leftover.. ;)

File: tests/e2e/wdio.conf.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+const __dirname = url.fileURLToPath(new URL('.', import.meta.url));
+export function setupTestPage() {
+  const file = readFileSync(path.join(__dirname, 'page.html'), 'utf8');
+  const server = createServer((req, res) => {
+    res.writeHead(200, { 'Content-Type': 'text/html' });
+    res.end(file);
+  });
+
+  // starts a simple http server locally on port 6789
+  server.listen(PAGE_PORT, '127.0.0.1', () => {
```
Comment (chrmod) [inline tests/e2e/wdio.conf.js:61]: we should close the server after the test

File: tests/e2e/wdio.update.conf.js [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+  existsSync,
+} from 'node:fs';
+import { execSync } from 'node:child_process';
+import { $, expect } from '@wdio/globals';
+
+import { getExtensionPageURL } from './utils.js';
+
+import * as wdio from './wdio.conf.js';
+
+export const config = {
```
Comment (chrmod) [inline tests/e2e/wdio.update.conf.js:27]: maybe add a comment on what this config is doing

File: tests/e2e/wdio.conf.js (hunk 3 above)
Comment (smalluban) [reply tests/e2e/wdio.conf.js:61]: This server is supposed to run during the whole testing process. It is set up only once and cleared out by the Nodejs itself. Look a the nodejs.org example from the home page - it does not closes itself too ;)

**Review by chrmod — APPROVED** [review]

**Pull request merged**